src/tools/compiletest/src/runtest.rs

```diff
@@ -242,15 +243,23 @@ pub fn compute_stamp_hash(config: &Config) -> String {
     let mut hash = DefaultHasher::new();
     config.stage_id.hash(&mut hash);
 
-    if config.mode == DebugInfoGdb || config.mode == DebugInfoBoth {
+    if config.mode == DebugInfoCdb {
+        match config.cdb {
+            None => env::var_os("ProgramFiles(x86)").hash(&mut hash),
```

Comment:
Although the `gdb` section does this, I think this is probably fine to just ignore and do nothing in this branch? It seems like this would probably want to just be `config.cdb.hash(&mut hash)`.

(I don't think 100% fidelity is really that necessary here and being simple is probably the best way to go)

Reply:
Works for me.